modules/dxp/apps/analytics/analytics-reports-web/src/main/java/com/liferay/analytics/reports/web/internal/model/SocialTrafficChannelImpl.java

```diff
@@ -37,16 +40,26 @@
 	public SocialTrafficChannelImpl(boolean error) {
 		_error = error;
 
-		_referringSocialMedia = Collections.emptyList();
+		_referringSocialMediaList = Collections.emptyList();
```

Comment:
Why should we change this name?

Reply:
Maybe referringSocialMedias?

Reply:
Media is already a plural, and normally the `List` suffix is added in this case. As happens with [List<CountrySearchKeywords> _countrySearchKeywordsList](https://github.com/liferay/liferay-portal/blob/master/modules/dxp/apps/analytics/analytics-reports-web/src/main/java/com/liferay/analytics/reports/web/internal/model/TrafficSource.java#L177). I prefer to use the one I proposed but I'm ok with your suggestion, actually was already used here [_getAdaptiveMedias](https://github.com/liferay/liferay-portal/blob/master/modules/apps/adaptive-media/adaptive-media-image-impl/src/main/java/com/liferay/adaptive/media/image/internal/media/query/MediaQueryProviderImpl.java#L141)